src/elements/Grid/index.js

```diff
@@ -67,49 +67,49 @@ export const getOffset = (colNumber, size) => {
   return `margin-left: ${width + grid[size].type};`;
 };
 
+export const getContainerMarginString = size => {
+  if (grid[size].gutter === 'auto') {
+    return `margin: 0 auto;`;
+  }
+  return `margin: 0 ${grid[size].gutter + grid[size].type};`;
+};
+
+export const getContainerWidthString = size => {
+  if (Object.prototype.hasOwnProperty.call(grid[size], 'width')) {
```

Comment:
[KISS](https://en.wikipedia.org/wiki/KISS_principle)©
```suggestion
  if (grid[size].width)) {
```